src/test/java/org/takes/rq/multipart/RqMultipartBaseTest.java

```diff
@@ -150,301 +152,17 @@ public void throwsExceptionOnNoClosingBoundaryFound() throws IOException {
-            Matchers.notNullValue()
-        );
-        multi.part("foo").iterator().next().body().close();
-        MatcherAssert.assertThat(
-            multi.part("bar").iterator().next(),
-            Matchers.notNullValue()
-        );
-        multi.part("bar").iterator().next().body().close();
-    }
-
-    /**
-     * RqMultipart.Fake can return empty iterator on invalid part request.
-     * @throws IOException If some problem inside
-     */
-    @Test
-    public void returnsEmptyIteratorOnInvalidPartRequest() throws IOException {
-        final String body = "443 N Wolfe Rd, Sunnyvale, CA 94085";
-        final RqMultipart multi = new RqMultipart.Fake(
-            new RqFake(),
-            new RqWithHeaders(
-                new RqFake("", "", body),
-                RqMultipartTest.contentLengthHeader(
-                    (long) body.getBytes().length
-                ),
-                RqMultipartTest.contentDispositionHeader(
-                    "form-data; name=\"t5\""
-                )
-            ),
-            new RqWithHeaders(
-                new RqFake("", "", ""),
-                RqMultipartTest.contentLengthHeader(0L),
-                RqMultipartTest.contentDispositionHeader(
-                    "form-data; name=\"data\"; filename=\"a.zip\""
-                )
-            )
-        );
-        MatcherAssert.assertThat(
-            multi.part("fake").iterator().hasNext(),
-            Matchers.is(false)
-        );
-        multi.body().close();
-    }
-
-    /**
-     * RqMultipart.Fake can return correct name set.
-     * @throws IOException If some problem inside
-     */
-    @Test
-    public void returnsCorrectNamesSet() throws IOException {
-        final String body = "441 N Wolfe Rd, Sunnyvale, CA 94085";
-        final RqMultipart multi = new RqMultipart.Fake(
-            new RqFake(),
-            new RqWithHeaders(
-                new RqFake("", "", body),
-                RqMultipartTest.contentLengthHeader(
-                    (long) body.getBytes().length
-                ),
-                RqMultipartTest.contentDispositionHeader(
-                    "form-data; name=\"address\""
-                )
-            ),
-            new RqWithHeaders(
-                new RqFake("", "", ""),
-                RqMultipartTest.contentLengthHeader(0L),
-                RqMultipartTest.contentDispositionHeader(
-                    "form-data; name=\"data\"; filename=\"a.bin\""
-                )
-            )
-        );
-        try {
-            MatcherAssert.assertThat(
-                multi.names(),
-                Matchers.<Iterable<String>>equalTo(
-                    new HashSet<String>(Arrays.asList("address", "data"))
-                )
-            );
-        } finally {
-            multi.body().close();
-        }
-    }
-
-    /**
-     * RqMultipart.Base can return correct part length.
+     * RqMultipartBase can return correct part length.
```

Comment:
@essobedo does this test really check `RqMultipartBase`? I think it's more for `RqMultipartSmart`